app/main.py

```diff
@@ -1,3 +1,72 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km):
+        self.km = km
+    
+    def __str__(self):
+        return f"Distance: {self.km} kilometers."
+    
+    def __repr__(self):
+        return f"Distance(km={self.distance})"
+    
+    def __add__(self, other):
+        if not isinstance(other, Distance):
+            raise TypeError(f"Wrong type(s) for +: 'Distance' and {type(other)}")
+        return Distance(
+            km=self.km + other.km
+        )
+    
+    def __iadd__(self, other):
+        if not isinstance(other, Distance):
+            raise TypeError("Can only add Distance or numeric types.")
+        
+        self.km += other
+        
+        return self
+        
+    
+    def __mul__(self, other):
+        if not isinstance(other, Distance):
+            raise TypeError("Can only multiply Distance or numeric types.")
+        
+        self.km *= other        
+        
+        return self
+    
+    def __truediv__(self, other):
+        if not isinstance(other, Distance):
+            raise TypeError("Can only divide Distance or numeric types.")
+        
+        self.km /= other
+        round(self.km, 2)
```

Comment:
The `round` function is not modifying `self.km` as intended. Assign the result of `round(self.km, 2)` back to `self.km` to ensure the value is correctly rounded.